src/content/security/index.md

```diff
@@ -0,0 +1,268 @@
+- [KeePass](https://keepass.info/)
+- [LastPass](https://www.lastpass.com/)
+- [1Password](https://1password.com/)
+
+### Use Two-Factor Authentication {#two-factor-authentication}
+
+To prove you are you there are different unique proofs that can be used for authentication. These are known as factors and the three main factors are:
+
+- Something you know (your password)
+- Something you are (such as a fingerprint or iris scanner)
+- Something you own (a security key)
+
+Using **Two-Factor Authentication (2FA)** provides an additional *security factor* for your online accounts so that knowing your password alone (something you know) is not enough to access an account. Most commonly, the second factor is a randomised 6-digit code (something you own) that you can access through an authenticator app such as Google Authenticator or Authy. These work as a "something you own" factor because the seed is stored on your device.
+
+<InfoBanner emoji=":lock:">
+  <div>
+    Note: Using SMS-based 2FA is susceptible to 
+    <a href="https://www.vice.com/en/article/3kx4ej/sim-jacking-mobile-phone-fraud">
+      SIM jacking
+    </a>
+     and is not secure. For the best security, use a service like{" "}
+    <a href="https://mashable.com/article/how-to-set-up-google-authenticator">
+      Google Authenticator
+    </a>
+     or <a href="https://authy.com/">Authy</a>.
+  </div>
+</InfoBanner>
+
+#### Security keys {#security-keys}
+
+For those who want to take the next step in 2FA, consider using a security key. Security keys are physical hardware authentication devices that work in the same way as authenticator apps. Using a security key is the most secure way to 2FA.
+
+Watch more on the 2FA:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/m8jlnZuV1i4?start=3479&end=3875" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Uninstall browser extentions {#uninstall-browser-extentions}
+
+Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.
+
+#### Stay safe by: {#browser-extention-safety}
+
+- Only install browser extensions from trusted sources
+- Removing unused browser extensions
+- Install Chrome extensions locally to stop auto-updating (Advanced)
+
+[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)
+
+<Divider />
+
+## Crypto Security 101 {#crypto-security}
+
+### Upgrade your knowledge {#upgrade-your-knowledge}
```

Comment:
"Level-up"? Small thought, but personally always pulled in more then this terminology is used
```suggestion
### Level up your knowledge {#level-up-your-knowledge}
```
Feel free to ignore =)